pointers/pointers.c

```diff
@@ -21,7 +23,16 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+    int len = sizeof(str) / sizeof(char);
```

Comment:
Same comment as mentioned before.

Reply:
It doesn't look like your function even needs this `len` variable anymore. Fire it into the sun and resolve this comment 👍